<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1156</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1121</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="btnEjercicio1">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>130</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Banner Semibold</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEjercicio2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>260</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Banner Semibold</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEjercicio3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>380</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Banner Semibold</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="RETURNMENU">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>570</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar al Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 1</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="PanelPiramide">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>661</width>
        <height>491</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="RETURNMENU_2">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>580</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar al Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejercicio 2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NumberCard">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>16</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>501</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Banner Semibold</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Porfavor digite los numeros de su tarjeta, sin espacios.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>200</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Verificar Tarjeta</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LineValidez">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Banner Semibold</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Su tarjeta es:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="RETURNMENU_3">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>570</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar al Menu</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="valorIngresado">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese Letra o Binario a convertir:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rtConversion">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>280</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>250</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Resultado de Conversion:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BTNtoBinario">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Letra a Binario</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BTNtoHexa">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>320</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Letra a Hexadecimal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BTNtoDecimal">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Letra a Decimal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BTNBinarioLT">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Binario a Letra</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BTNDecimalLt">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>280</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Decimal a Letra</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BTNHexaLt">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>330</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Hexadecimal a Letra</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>650</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SALIR DEL PROGRAMA</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1156</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
